apps/condo/domains/property/schema/ExportPropertiesToExcelService.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Generated by `createservice property.ExportPropertiesToExcelService --type queries`
+ */
+
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const access = require('@condo/domains/property/access/ExportPropertiesToExcelService')
+const { Property: PropertyAPI } = require('@condo/domains/property/utils/serverSchema')
+const { EMPTY_DATA_EXPORT_ERROR } = require('@condo/domains/common/constants/errors')
+const moment = require('moment')
+const { createExportFile } = require('@condo/domains/common/utils/createExportFile')
+
+
+const CHUNK_SIZE = 50
+
+
+const ExportPropertiesToExcelService = new GQLCustomSchema('ExportPropertiesToExcelService', {
+    types: [
+        { 
+            access: true,
+            type: 'input ExportPropertiesToExcelInput { where: PropertyWhereInput!, sortBy: [SortPropertiesBy!] }',
+        },
+        {
+            access: true,
+            type: 'type ExportPropertiesToExcelOutput { status: String!, linkToFile: String!  }',
+        },
+    ],
+    
+    queries: [
+        {
+            access: access.canExportPropertiesToExcel,
+            schema: 'executeExportPropertiesToExcel (data: ExportPropertiesToExcelInput!): ExportPropertiesToExcelOutput',
+            resolver: async (parent, args, context, info, extra = {}) => {
+                const { where, sortBy } = args.data
+                let skip = 0
+                let maxCount = 1000
```

Comment:
Is this artificial limit to export? I guess you  should move it to consts like `CHUNK_SIZE` and name something like `MAX_QUERIES_COUNT`

Reply:
renamed to iterationsLimit